public/Sjofn/practice/sjofn4.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Practice 4</title>
+</head>
+<body>
+</body>
+<script>
+    function getDatetime() {
+        console.log(new Date());
+    }
+    function getPowFive(z) {
```

Comment:
Please, leave an emply line between functions. 
It is a good practice and makes the code more readable.